src/runtime_src/hip/api/hip_memory.cpp

```diff
@@ -193,6 +194,22 @@ namespace xrt::core::hip
     };
   }
 
+  static void
+  hip_memcpy_async(void* dst, const void* src, size_t size, hipMemcpyKind kind, hipStream_t stream)
+  {
+    throw_invalid_value_if(!dst, "dst is nullptr.");
+    throw_invalid_value_if(!src, "src is nullptr.");
+
+    auto hip_stream = get_stream(stream);
+    throw_invalid_value_if(!hip_stream, "Invalid stream handle.");
+
+    // ptr to a xrt::core::hip::command object could be shared between global command_cache and stream::m_top_event::m_chain_of_commands of a stream object
```

Comment:
Lets split this into lines approximately 80 chars long

Reply:
fixed.